Form-Controls/index.html

```diff
@@ -9,19 +9,67 @@
   <link rel="stylesheet" href="styles.css">
 </head>
 <body>
+
   <header>
     <h1>Product Pick</h1>
   </header>
   <main>
+    <h2 class="para">Please fill in the order form to complete your order</h2>
+
   <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
-  </form>
+  <div class="form">
+    <label for="fname">First Name:</label><br>
+    <input type="text" id="fname" name="fname" pattern="[A-Za-z]+" title="Only letters are allowed" required/>
```

Comment:
Nice use of `pattern`! One thing to consider -- names vary widely and may not include characters beyond ASCII or not necessarily be easily splittable into first/last names.

See these blog posts for more assumptions that aren't true about names:
- [What's in a name](https://dev.to/carlymho/whats-in-a-name-validation-4b41)
- [Falsehoods Programmers Believe About Names](https://www.kalzumeus.com/2010/06/17/falsehoods-programmers-believe-about-names/)

Any thoughts on this, e.g. changing the pattern and/or combining the name fields?

Reply:
should i do full name insread of first name and last name?

Reply:
fixed